### ISSUE TITLE
BrowscapUpdater frequently times out

### ISSUE THREAD

**Issue by ianchadwick** [browscap/browscap-php#165]
Running either the update or fetch command with Full_PHP_BrowscapINI remote-file times out frequently.

`browscap:update --remote-file Full_PHP_BrowscapINI`

>  [GuzzleHttp\Exception\ConnectException]                                                                                                          
>   cURL error 28: Operation timed out after 5000 milliseconds with 18367976 bytes received (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)

**Comment by cbj4074** [comment]
Just updated from `3.0.0-rc.1` to `3.0.0` and am now seeing the same thing:

```
vendor/bin/browscap-php browscap:update
[2016-07-16 13:29:26] started updating cache with remote file {"memory_peak_usage":"4 MB"}
[2016-07-16 13:29:26] there is no cached version available, please update from remote {"memory_peak_usage":"4 MB"}


  [GuzzleHttp\Exception\ConnectException]
  cURL error 28: Connection timed out after 5000 milliseconds (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)


browscap:update [-r|--remote-file [REMOTE-FILE]] [--no-backup] [-d|--debug] [-c|--cache [CACHE]]
```

**Comment by jkns** [comment]
I've just tried to setup Browscap for the first time. I kept getting the timeout issue. Updating to 30 didn't work. Ended up changing it to 60 before it downloaded.

**Comment by asgrim** [comment]
Closed by #167
